bench/config.yaml

```diff
@@ -76,7 +76,7 @@ versions:
 
 
 # - 1.8.0.0
-# - upstream: origin/master
+- upstream: origin/master
```

Comment:
Yeah, I disabled it temporarily when merging the benchmark suite PR because `upstream/master` lacked some fixes that were required to run it. Thanks for enabling it!